test/node-rpc-test.js

```diff
@@ -68,11 +70,93 @@ describe('RPC', function() {
     assert.deepEqual(info.localservicenames, ['NETWORK', 'WITNESS']);
   });
 
+  it('should rpc getblockhash', async () => {
+    const info = await nclient.execute('getblockhash', [node.chain.tip.height]);
+    assert.strictEqual(util.revHex(node.chain.tip.hash), info);
+  });
+
+  describe('Blockchain', function () {
+    it('should rpc getchaintips', async () => {
+      const info = await nclient.execute('getchaintips', []);
+      assert.strictEqual(info.length, 1);
+      assert.strictEqual(util.revHex(node.chain.tip.hash), info[0].hash);
+    });
+
+    it('should rpc getchaintips for chain fork', async () => {
+      const {genesis} = node.network;
+      let entry = await node.chain.getEntry(genesis.height);
+     // extnding chain1 from genesis.
```

Comment:
Right!, but I thought of doing it this way. As some people believe that genesis height is `1`. So, I thought it would be better just to use `genesis.height`.